core/src/main/java/nu/yona/server/subscriptions/service/BuddyDisconnectMessageDTO.java

```diff
@@ -120,7 +120,7 @@ public MessageActionDTO handleAction(UserDTO actingUser, Message messageEntity,
 		private MessageActionDTO handleAction_Process(UserDTO actingUser, BuddyDisconnectMessage messageEntity,
 				MessageActionDTO requestPayload)
 		{
-			buddyService.removeBuddyAfterBuddyRemovedConnection(actingUser.getID(), messageEntity.getUserID());
+			buddyService.removeBuddyAfterBuddyRemovedConnection(actingUser.getID(), messageEntity.getSenderUserID());
```

Comment:
This process action is no longer needed if `messageEntity.getReason() == DropBuddyReason.USER_ACCOUNT_DELETED`, is it? I think we could remove the action in this case?